CMakeLists.txt

```diff
@@ -711,7 +691,6 @@ endif ()
 if (CMAKE_SYSTEM_NAME STREQUAL "Emscripten")
   set (H5_HAVE_EMBEDDED_LIBINFO 0)
 else ()
-  option (HDF5_ENABLE_EMBEDDED_LIBINFO "Embed library info into executables" ON)
```

Comment:
Moving this option to the top-level build file kind of changes the semantics here since this was previously inside a conditional.

Reply:
It wasn't a controlling conditional jus a do do it for emscripten